log/operation_manager.go

```diff
@@ -225,22 +228,14 @@ func (o *OperationManager) masterFor(ctx context.Context, allIDs []int64) ([]int
 	// Synchronize the set of log IDs with those we are tracking mastership for.
 	for _, logID := range allStringIDs {
 		knownLogs.Set(1, logID)
-		if o.electionRunner[logID] != nil {
+		if o.runnerCancels[logID] != nil {
```

Comment:
Not sure whether this or `if _, ok := ` is preferred style, or maybe doesn't matter.

Reply:
In this case it is equivalent.